library/core/src/intrinsics.rs

```diff
@@ -2574,14 +2574,9 @@ pub(crate) fn is_nonoverlapping<T>(src: *const T, dst: *const T, count: usize) -
 ///
 /// For regions of memory which might overlap, use [`copy`] instead.
 ///
-/// `copy_nonoverlapping` is semantically equivalent to C's [`memcpy`], but
-/// with the argument order swapped.
```

Comment:
Is it worth saying that this is still the same as `memcpy` for non-zero-sized copies, or that it is similar to `memcpy`, or something like that?

Reply:
Many other programmers come from C though and it is helpful for them. :shrug: In particular since it is almost-but-not-quite the same.

Reply:
The problem is that we have no policy here, so discussing the deviations in detail invites "why not talk about this similar function from `{lang}` until we have 10 different languages being discussed in what is supposed to be the documentation for a single function. T-libs has previously resolved to avoid this direction until a draft policy is proposed.

Reply:
Also in actual cases, colleagues have noticed things like `write_bytes` saying "it's like memset" has caused programmers to **stop reading there**, and not even *think* about the disclaimer "...but it writes for this larger space", causing people who intended to "memset" a single page to instead thrash 16 pages because they didn't think about pointer type.